src/loaders/OMETiffLoader.js

```diff
@@ -47,7 +47,7 @@ export default class OMETiffLoader {
     // This works with the current bioformats6 pipeline.
     // Related to: https://github.com/hubmapconsortium/vitessce-image-viewer/issues/144
     this.numLevels =
-      this.omexml.getNumberOfImages() || (SubIFDs && SubIFDs.length - 1);
+      this.omexml.getNumberOfImages() || (SubIFDs && SubIFDs.length);
```

Comment:
I've been using JavaScript [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) in these situations lately. 

```suggestion
       this.omexml.getNumberOfImages() || SubIFDs?.length;
```